src/controller/task.ts

```diff
@@ -0,0 +1,75 @@
+import { Request, Response, NextFunction } from 'express'
+import TaskModel, { Task } from '../model/task'
+
+export default {
+  async createTask (req: Request, res: Response, next: NextFunction): Promise<void> {
+    try {
+      const { title, description, importance, status } = req.body
+      const newTask: Task = await TaskModel.create({
+        title,
+        description,
+        importance,
+        status
+      })
+      res.status(201).send(newTask)
+    } catch (err) {
+      next(err)
+    }
+  },
+
+  async getAllTasks (req: Request, res: Response, next: NextFunction): Promise<void> {
+    try {
+      const tasks: Task[] = await TaskModel.find()
```

Comment:
This method should support pagination. 

Query parameters: offset = 0, limit=20 (max 100)

So, the first request should return list of 20 docs.  (default)
If I send request with: offset=20 -> I should get next 20 docs, skipping the first 20. 

Response should looks like this: 

```js
{
  'tasks': [],
  'limit': 20,
  'offset': 0,
  'total': 200 // total could of docs for this request
}
```